procedures/katello/release.md.erb

```diff
@@ -14,42 +14,54 @@
 - [ ] Do Cherry-picks: Clone [tool_belt](https://github.com/theforeman/tool_belt)
   - [ ] If any minor versions of Katello have been added to Redmine since the last cherry-pick, make sure to include them in `prior_releases` in configs/katello/<%= short_version %>.yaml
   - [ ] Run `./tools.rb setup-environment configs/katello/<%= short_version %>.yaml`
-  - [ ] Run `GITHUB_ACCESS_TOKEN=<secret> ./tools.rb cherry-picks --version <%= full_version %> configs/katello/<%= short_version %>.yaml`
+  - [ ] Run `GITHUB_ACCESS_TOKEN=<secret> PROJECT=katello VERSION=<%= short_version %> ./tools.rb cherry-picks configs/katello/<%= short_version %>.yaml`
   - [ ] Open a PR in Katello release branch.  Make sure the PR name starts with [CP] to prevent our automations from adding it to Redmine issues.
   - [ ] Using `git cherry-pick -x` as needed, verify tickets in the cherry_picks_<%= full_version %> file are accounted for, or additionally cherry-pick them.  Recommended: Do this in tool_belt's checkout of Katello, in `repos/katello/<%= full_version %>/katello`. This way when you run cherry-picks again, tool_belt will be aware of any picks already completed.
   - [ ] For any cherry-picks that are not needed (including Redmine trackers) you can add them to the `:ignores:` section of `tool_belt` in `configs/katello/<%= short_version %>.yaml`
+- [ ] Update Katello Transifex translations:
+  - [ ] Create a Transifex account and join the Foreman team
+  - [ ] Spin up a Foreman and Katello installation
+  - [ ] [Configure the Transifex client](https://projects.theforeman.org/projects/foreman/wiki/Translating#How-to-pull-translations)
+  - [ ] [Install grunt](https://github.com/Katello/katello/blob/master/engines/bastion_katello/README.md)
+  - [ ] `grunt i18n:extract` in the katello/engines/bastion_katello directory
+  - [ ] `tx pull --minimum-perc 50 --all` in the katello directory
```

Comment:
Since https://github.com/Katello/katello/commit/2607abb55b6154c8f2763dc444b418ce48d3eb06 `Makefile` does this when running `tx-pull`
```suggestion
```